robot/framesystem/framesystem.go

```diff
@@ -203,6 +206,36 @@ func (svc *frameSystemService) TransformPose(
 	return pose, nil
 }
 
+// TransformPointCloud applies the same pose offset to each point in a single pointcloud and returns the transformed point cloud.
+// if destination string is empty, defaults to transforming to the world frame.
+// Do not move the robot between the generation of the initial pointcloud and the receipt
+// of the transformed pointcloud because that will make the transformations inaccurate.
+func (svc *frameSystemService) TransformPointCloud(ctx context.Context, srcpc pointcloud.PointCloud, srcName, dstName string,
+) (pointcloud.PointCloud, error) {
+	// get FrameSystem from Robot
+	fs, err := RobotFrameSystem(ctx, svc.r, nil)
+	if err != nil {
+		return nil, err
+	}
+	// initialize frame system inputs
+	inputs, _, err := RobotFsCurrentInputs(ctx, svc.r, fs)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I'm wondering if this is still the way to initialize the robot framesystem now to grab the latest info?

Reply:
Actually, I'm gonna edit this to use transformPose